### ISSUE TITLE
meta generator displays on 404 pages

### ISSUE THREAD

**Issue by SerGun78** [templaza/astroid-framework#556]
When you enable the meta tag removal plugin in Astroyd, it is still shown on the error pages.
![opera_NOFPoqhmcY](https://github.com/templaza/astroid-framework/assets/95705148/da1d17a4-39d9-4ad3-90df-6b2db3e2b48c)
![oi2Uckawxb](https://github.com/templaza/astroid-framework/assets/95705148/3446892e-7fd7-4f51-8b68-53cc28633f3c)

**Comment by salahjaafar** [comment]
Please publish the displayed error so that we can better understand the situation and find an appropriate solution.

**Comment by sonvnn** [comment]
Thanks! I will check it. May be a bit slowly because I am in Lunar New Year holiday.

**Comment by sonvnn** [comment]
May you send me your admin account via sonlv@templaza.com? I will help you check it.

**Comment by SerGun78** [comment]
Sonny, hi. Unfortunately, I cannot give access. Why give it? You can check it yourself. Enable hiding the Joomla meta tag in the Astroyd plugin, and then go to the error page and check its code.

**Comment by Giorgi625** [comment]
It is normal practic to give access to developer to check it, because for example on my website when I enable in plugin to hide generator it works just fine. so most probably there is some problem on your website so he should have possibility to check what is wrong.

**Comment by SerGun78** [comment]
Giorgi625, can I have a screenshot from your error page?

**Comment by SerGun78** [comment]
But what does the error page itself look like in the frontend?

**Comment by Giorgi625** [comment]
Yes I missed that you were reporting problem for error page, but as you can see I do not have problem on error page either.

![noname](https://github.com/templaza/astroid-framework/assets/16633699/2c5af08a-91f5-463c-b75f-126654d8aaff)

**Comment by SerGun78** [comment]
Astroid 3.0.10?
I found this error on 3.0.9.

**Comment by Giorgi625** [comment]
Yep 
Framework Version: 3.0.10
Template Two Version: 1.0.2

**Comment by SerGun78** [comment]
All clear. So Sonny made changes in 3.0.10, but it's not in the version history.

**Comment by Giorgi625** [comment]
I checked on another site with 3.0.9 and I do not see generator tag there too.

**Comment by SerGun78** [comment]
Now I have updated to 3.0.10. Now everything is fine, there is no joomla meta tag in the error page code.

**Comment by SerGun78** [comment]
> I checked on another site with 3.0.9 and I do not see generator tag there too.

check on template "one"

**Comment by Giorgi625** [comment]
yes I see meta generator on template one when in plugin it is disabled.

**Comment by SerGun78** [comment]
it means there was an error.
Sonia cleaned up heavily in error.php
the meta tag has disappeared, but visually the error page itself is now different from the working pages of the site. You can't do that.

**Comment by felixkat** [comment]
Closing as the original appears to have been solved.
